mock/docs/mock.1

```diff
@@ -137,6 +149,13 @@ Do a yum install PACKAGE inside the chroot. No 'clean' is performed.
 \fB\-\-installdeps\fP
 Find out deps for SRPM or RPM, and do a yum install to put them in the chroot. No 'clean' is performed
 .TP
+\fB\-\-isolated\-build\-config \fImock\-build\-environment.json\fR \fIREPO\fR \fISRPM\fR
+Perform an isolated RPM build from SRPM (= an offline one, without the need to
+access Internet).  After running the \fB\-\-calculate\-build\-dependencies\fR
+first to get the \fImock\-build\-environment.json\fR file, and then running the
+\fImock\-isolated\-repo(1)\fR helper for generating \fIREPO\fR, Mock has all the
+necessary info for building RPMs from the \fISRPM\fR fully offline.
+.TP
```

Comment:
I would explicitely state that the file is filename relative to PWD. To not repeat history with -r option.

Reply:
It may be any file, not necessary relative to CWD

Reply:
I added notes like `Evaluate  all  the SRPM (= file name, path on your system)`.